### PR TITLE
Connection Drawer

### PR DESCRIPTION
**Pull request by lspdrz**
DO NOT MERGE UNTIL https://github.com/Ameelio/connect-api-nest/pull/489 is merged!

[sc-1389]

https://user-images.githubusercontent.com/9705113/142228544-47770f40-ebe5-4b70-8e64-2000eaa37ca9.mov

### CODE REVIEW

**Review by wittejm — APPROVED** [review]

File: src/components/Profile/Connections/ConnectionItem.tsx [hunk 1]
```diff
@@ -51,10 +60,64 @@ const ConnectItemButton = ({
...
+  return (
+    <>
+      <Descriptions.Item label='Disabled by' labelStyle={{
+        paddingLeft: '25px',
+        color: '#aaaaaa'
+      }}>
+        {<StaffCell staff={restriction.createdBy}/>}
+      </Descriptions.Item>
+      <Descriptions.Item label='Disabled on' labelStyle={{
+        paddingLeft: '25px',
```
Comment (gabrielsaruhashi) [inline src/components/Profile/Connections/ConnectionItem.tsx]: could we turn this into a component?
seems like we're repeating the label style on every single item

File: src/components/Profile/Connections/ConnectionItem.tsx [hunk 2]
```diff
@@ -51,10 +60,64 @@ const ConnectItemButton = ({
   }
 };
 
+const getRestrictionDetails = (restriction: ConnectionRestriction) => {
+  return (
+    <>
+      <Descriptions.Item label='Disabled by' labelStyle={{
```
[the remark this answers is not in the document]
Comment (lspdrz) [reply src/components/Profile/Connections/ConnectionItem.tsx]: Great idea it's done

File: src/components/Profile/Connections/ConnectionItem.tsx (hunk 1 above)
Comment (lspdrz) [reply src/components/Profile/Connections/ConnectionItem.tsx]: See the most recent commit from Jordan's suggestion

File: src/components/Profile/Connections/ConnectionItem.tsx [hunk 3]
```diff
@@ -51,10 +60,50 @@ const ConnectItemButton = ({
...
+      paddingLeft: '25px',
+      color: '#aaaaaa'
+    }}>
+      {content}
+    </Descriptions.Item>
+  ))
+}
+
 const ConnectionItem: React.FC<Props> = ({ connection, type, onUpdate }) => {
+  const { loading, error, data } = useQuery(GetConnectionConnectionRestrictionsDocument, {
```
Comment (gabrielsaruhashi) [inline src/components/Profile/Connections/ConnectionItem.tsx:87]: @lspdrz  u need to hide this behind a feature flag

**Pull request merged**